spyderlib/userconfig.py

```diff
@@ -251,7 +253,13 @@ def load_from_ini(self):
         try:
             if PY2:
                 # Python 2
-                self.read(self.filename())
+                fname = self.filename()
+                if osp.isfile(fname):
+                    try:
+                        with codecs.open(fname, encoding='utf-8') as configfile:
+                            self.readfp(configfile)

```

Comment:
read() expects filename and does file open itself, readfp() expects open file handle